rootfs/scheduler/mock.py

```diff
@@ -450,6 +450,7 @@ def manage_replicasets(deployment, url):
 
     # get RS url
     rs_url = url.replace('_deployments_', '_replicasets_')
+    rs_url = rs_url.replace('apis_extensions_v1beta1_', 'apis_apps_v1_')
```

Comment:
Should this be dependent on Kubernetes version?

Reply:
Yes, it should. If it was code in the scheduler or the code that creates replicaset or deployments, but here it only mocks the routes for the test-cluster created in the mocker.

Reply:
Addressed in d8e65a9. Thanks a lot for the comments!